quickstarts/snyk/config.yml

```diff
@@ -0,0 +1,49 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: snyk
+
+# Displayed in the UI (required)
+title: Snyk
+
+# Long-form description of the quickstart (required)
+description: |
+  The Snyk quickstart allows you to send application security vulnerabilities of your services into New Relic.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Application security vulnerabilities from within New Relic with this integration by Snyk.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Harry Kimpel
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - Snyk
+  - Application Security
+  - AppSec
+  - DevSecOps
+  - Vulnerabilities
```

Comment:
```suggestion
  - Vulnerabilities
  - newrelic partner
  - azure
  - apm
  - security
```

Reply:
@jcountsNR, to get it into the "Application monitoring" category, you'll want to use the `language agent` keyword. However, this isn't a language agent, so I don't think we want to include it in that category. Just my 2c, feel free to override!

Reply:
> @harrykimpel looks like the main issue here is the conflict with the .gitignore file. It might be best to just remove the changes there. Other than that it looks good to me! I added a few suggestions like adding to the description and adding a few more keywords. `apm` is very important here because that is how it will be categorized under `application monitoring`

@jcountsNR I just submitted an update to the PR